.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules/
+.DS_Store
+*.swp
+npm-debug.log
```

Comment:
I'm trying to phase these out from my repos, instead encouraging global gitignore files for this stuff.